ec/acc/deployment_basic_test.go

```diff
@@ -84,6 +80,29 @@ func TestAccDeployment_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resName, "apm.0.topology.0.config.0.debug_enabled", "false"),
 				),
 			},
+		},
+	})
+}
+
+func TestAccDeployment_basic_config(t *testing.T) {
+	resName := "ec_deployment.basic"
+	randomName := prefix + acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
+	startCfg := "testdata/deployment_basic.tf"
+	topologyConfig := "testdata/deployment_basic_topology_config.tf"
+	topConfig := "testdata/deployment_basic_top_config.tf"
```

Comment:
Maybe this is a good chance to add a suffix as `_2`, `_3`, We haven't discussed this as something we should do but I think it makes searching for the test fixture files easier.

Reply:
Sure! makes sense